### ISSUE TITLE
"indices" as disappeared on some of my clusters

### ISSUE THREAD

**Issue by SosthenG** [AntonShuvaev/elasticsearch4idea#99]
Hi,

Since the last update I noticed that the "indices" node as disappeared but only for some of my clusters.
I can still send queries to the indices using the query console so the connection is ok.
The only common point I can find is that the two clusters missing the "indices" node are read-only remote clusters in ES 7.*

![Screenshot of the missing nodes](https://user-images.githubusercontent.com/22033159/224041704-a47190e9-47ba-4915-b2cc-7652008993f4.png)

Thanks.

**Comment by AntonShuvaev** [comment]
Hi, thanks for reporting. I'll look into it.

**Comment by AntonShuvaev** [comment]
Could you execute `GET /_cat/indices?format=json` and share what property names are present in index object?

**Comment by SosthenG** [comment]
Sure, here is an example of the properties I get :
```
{
    "health": "green",
    "status": "open",
    "index": "offer_it_ri",
    "uuid": "5sOp4s7qTxmMG_ojlkZCfA",
    "pri": "1",
    "rep": "2",
    "docs.count": "3098802",
    "docs.deleted": "943259",
    "store.size": "477.7mb",
    "pri.store.size": "159.2mb"
  }
```

I forgot something the two clusters have in common: they are only accessible when I'm connected to a VPN (while the 3 others are always accessible). Not sure it's relevant but anyway.

**Comment by AntonShuvaev** [comment]
I released a fix, but I'm not 100% sure that it will fix the issue. So, I also added additional logging to the code. If the fix doesn't help, I'll need you to send me the stacktrace of the error.

To enable logging go to `Help | Diagnostic Tools | Debug Log Settings` and add `org.elasticsearch4idea:trace` to the textarea. Then reconnect to cluster to cause the error and find the stacktrace in `idea.log` file (`Help | Show Log in Finder`).

You can download the version with fix here: https://plugins.jetbrains.com/files/14512/305447/Elasticsearch-2023.1.1-223-signed.zip and install it manually or wait until it will be available in marketplace (it may take up to 2 business days).

**Comment by hwsong** [comment]
Even after installing the modified version, the same problem occurs.
In my case, indices are not visible when connecting using kubernetes port-forward.
Please note that the log is captured and attached as described above.
[idea.log](https://github.com/AntonShuvaev/elasticsearch4idea/files/10954126/idea.log)

**Comment by AntonShuvaev** [comment]
@hwsong Thanks for sharing the logs. I've released a new version with a fix. It will be available once it is approved.

**Comment by AntonShuvaev** [comment]
Fixed with 2023.1.2
